lib/src/cli_client.dart

```diff
@@ -16,29 +16,27 @@ import 'services/travis.dart' as travis;
 
 class CommandLineClient {
   final String projectDirectory;
-  final String packageRoot;
```

Comment:
Given how busted dart_coveralls is now, I think it's OK to leave it busted in the package-root case, IMO.  We can tackle that in a separate change if there are still users who need that.